small-talk/app/components/SidebarButton.js

```diff
@@ -1,27 +1,34 @@
 /**
- * SidebarButton.js is a specific button using the HoverButton component style that is intended to be used for the homepage
+ * SidebarButton.js is a specific button when it is in the nonexpanded state that is intended to be used for the layout
  * sidebar list of buttons. e.g. Game, Chat, Order Food, etc.
  * 
  * @param {string} redirect - the redirect path for the specific button
- * @param {string} imgSrc - the path for the image associated with this sidebar button
+ * @param {string} defaultImg - the path for the image associated with this sidebar button when it is not hovered
+ * @param {string} hoverImg - the path for the image associated with this sidebar button when it is hovered
  * @param {string} altText - the name of the button used for when the image cannot load (or other defined cases)
- * @param {string} width
- * @param {string} height
- * @param {string} marginLeft - width, height, and marginLeft define the custom sizes for the images to look good in the sidebar
- *                              these images should be resized to be more responsive and consistent down the line.
  */
-import React from 'react';
-import HoverButton from './HoverButton';
+import React, { useState } from 'react';
+import Link from "next/link";
 
-const SidebarButton = ({ redirect, imgSrc, altText, width, height, marginLeft }) => {
-    const handleRedirect = (path) => () => {
-        window.location.href = path; 
-    };
+const SidebarButton = ({ redirect, defaultImg, hoverImg, altText }) => {
+    const [isHovering, setIsHovered] = useState(false);
+    const onMouseEnter = () => setIsHovered(true);
+    const onMouseLeave = () => setIsHovered(false);
 
     return (
-        <HoverButton onClick={handleRedirect(redirect)}>
-            <img src={imgSrc} alt={altText} style={{ width, height, marginLeft }} />
-        </HoverButton>
+        <div 
+            className="bg-white h-[17.6%] m-0"
+            onMouseEnter={onMouseEnter}
+            onMouseLeave={onMouseLeave}
```

Comment:
Not sure what everyone else is seeing, but there's some glitchiness to the hovering effects here. It'll flip back and forth between two buttons super quick the first time I'm transitioning between two buttons. The hover effect is likely changing the 'hitbox' for the buttons and causing the browser to think we're selecting two buttons at the same time maybe? I think it's okay as is for MVP but might be something to consider for the future.

Reply:
Yeah, I intend to work on it probably after DP II is done just so I can get out some of the issues with it and forgot to add it to the issues mentioned in the PR, I was thinking maybe having a transition or slight delay so it isn't an instant switch and we can hopefully avoid that.